sdk/monitor/monitor-query/test/internal/unit/modelConverters.unittest.spec.ts

```diff
@@ -24,12 +24,7 @@ import {
   SerializerOptions
 } from "@azure/core-client";
 import { OperationTracingOptions } from "@azure/core-tracing";
-import {
-  Durations,
-  ListMetricDefinitionsOptions,
-  MetricsQueryOptions,
-  MetricsQueryResult
-} from "../../../src";
+import { Durations, ListMetricDefinitionsOptions, MetricsQueryOptions } from "../../../src";
```

Comment:
Where can I find the tests for the logic that compute status for failed or succeed or partial results

Reply:
oh I haven't written tests on those

Reply:
Created an issue - https://github.com/Azure/azure-sdk-for-js/issues/17502